src/BaseTest.php

```diff
@@ -79,10 +79,10 @@ protected function buildStream($data)
         if (defined('STREAM_FACTORY')) {
             $factoryClass = STREAM_FACTORY;
             $factory = new $factoryClass();
-            if ($factory instanceof \Http\Message\StreamFactory) {
+            if ($factory instanceof HttpPlugStreamFactory) {
```

Comment:
```suggestion
            if ($factory instanceof HttpplugStreamFactory) {
```

missing alias.

but i think i would prefer to not alias these classes and keep using FQN. that way it is very explicit what its about.